src/utils/awsConfig.js

```diff
@@ -0,0 +1,20 @@
+import { useSelector } from 'react-redux';
+
+const getAccountId = () => {
+  let accountID;
+  const environment = useSelector((state) => state.networkResources.environment);
+
+  if (environment === 'development') {
+    accountID = '000000000000';
+  } else {
+    accountID = '242905224710';
+  }
```

Comment:
don't do that -- we can't be hardcoding account id like this. You can use aws cli to get the account id, see this function that we use in the api:

<img width="493" alt="Screenshot 2022-06-27 at 10 54 09" src="https://user-images.githubusercontent.com/65414651/175900280-d73bd080-aaf1-4f46-acbd-d512c734b40c.png">